courses/machine_learning/deepdive2/structured/solutions/3c_bqml_dnn_babyweight.ipynb

```diff
@@ -0,0 +1,1152 @@
+{
```

Comment:
<ul><li>Use %%bigquery &lt;destination_var&gt; or <code>--destination_table</code>&nbsp; to store the evaluation results (either in a local variable or in a BQ table) so that we can retrieve them without recomputing them in the next cell</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/670/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Don't want to use extra storage and it is a pretty fast query.</p>